src/components/CartContextWrapper/CartContextWrapper.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+
+import { CartContext } from "../../util/context";
+
+export const CartContextWrapper = ({ children }) => {
+  const [cart, setCart] = useState([
+    {
+      id: 1,
+      unitPrice: 1200,
+      quantity: 1,
+      maxQuantity: 5,
```

Comment:
I thought we agreed on having only `id, price, and quantity` in the cart?

Reply:
i added maxQuantity to add a feature on the button where customer can not add more than the quantity. But i will remove it